ci/create_experment.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+'''
+Basic python script to create an experment directory on the fly from a given
+yaml file for the arguments to the two scripts below in ${HOMEgfs}/workflow
+where ${HOMEgfs} is specified within the input yaml file.
+
+${HOMEgfs}/workflow/setup_expt.py
+${HOMEgfs}/workflow/setup_xml.py
+'''
+
+import os, sys, socket
+import glob
+from pathlib import Path
+
+from pygw.yaml_file import YAMLFile
+from pygw.logger import Logger
+from pygw.executable import Executable
+
+from argparse import ArgumentParser, ArgumentDefaultsHelpFormatter
+
+from hosts import Host
+
+logger = Logger(level='DEBUG', colored_log=True)
+_here = Path.absolute(Path(__file__)).parents[0]
+_top  = Path.absolute(Path(__file__)).parents[1]
+
+def input_args():
+    """
+    Method to collect user arguments for `create_experment.py`
+    """
+    description = """
+        Single agument as a yaml file
+        """
```

Comment:
Please update to use numpy/reST format: https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard

Reply:
will do

Reply:
Nope. You don't have to repeat the function name, but you do need to flesh out the rest more. Collect them how? What is the return value? See both the numpy documentation I provided and https://github.com/NOAA-EMC/global-workflow/blob/develop/ush/python/pygfs/task/analysis.py

Also, `description` probably should just be a single-line string with a blank line separating it from the docblock.  I mistook it for part of the docblock the first time.

Reply:
ok

Reply:
This docblock is still not in numpy format. Needs to be something like this:
```
"""
Method to collect user arguments
 
Parses all of the command-line arguments with argparse and returns a namespace with the values. Currently the only argument is the path to a YAML file that contains the experiment information. That YAML file should contain the following:

<list parameters of YAML with descriptions>

Returns
--------
args:   Namespace
    Namespace containing all of the command-line arguments
"""
```

Reply:
ok, still got to get that right, thanks.

Reply:
So why was it not resolved, I had ''' instead of """?  I added a Return comment section incase that was the issue.

Reply:
Ok, I've asked to give more content on the workings of the YAML file itself.  I agree this needs some edification.

Reply:
Ok, pushed the updates for the doc-block, I hope this cuts the mustered this time.